lib/api.js

```diff
@@ -91,32 +93,23 @@ PlexAPI.prototype._request = function _request(relativeUrl, method, parseRespons
         url: url.parse(reqUrl),
         encoding: null,
         method: method || 'GET',
-        headers: {
+        headers: headers(this, {
             'Accept': 'application/json',
-            'X-Plex-Client-Identifier': self.getIdentifier(),
-            'X-Plex-Product': self.options.product,
-            'X-Plex-Version': self.options.version,
-            'X-Plex-Device': self.options.device,
-            'X-Plex-Device-Name': self.options.deviceName,
-            'X-Plex-Platform': self.options.platform,
-            'X-Plex-Platform-Version': self.options.platformVersion,
-            'X-Plex-Provides': 'controller'
-        }
+            'X-Plex-Token': this.authToken,
+            'X-Plex-Username': this.username
```

Comment:
Made `plex-api-headers` so that additional headers has to have `string` as their value for them to be added. It allows for the explicit if-statements below to be deleted.